templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>Create manufacturer:</h1>
```

Comment:
What about an **update**?

Reply:
Thanks! I've refactored and added changes!